src/proxy/grpc_app.go

```diff
@@ -33,16 +35,16 @@ type GrpcAppProxy struct {
 	server   *grpc.Server
 
 	timeout      time.Duration
-	new_clients  chan ClientStream
-	askings      map[xid.ID]chan *internal.ToServer_Answer
-	askings_sync sync.RWMutex
+	newClients  chan ClientStream
+	askings      map[xid.ID]chan *internal.ToServerAnswer
```

Comment:
Do not rename internal-package's entities: they are generated by protoc-gen-go from grpc.proto. The same below